tools/testbench/topology.c

```diff
@@ -369,23 +379,33 @@ static int tb_register_fileread(struct testbench_prm *tp,
 	tp->input_file_index++;
 
 	/* Set format from testbench command line*/
-	fileread.rate = tp->fs_in;
-	fileread.channels = tp->channels_in;
-	fileread.frame_fmt = tp->frame_fmt;
-	fileread.direction = dir;
-
-	/* Set type depending on direction */
-	fileread.comp.type = (dir == SOF_IPC_STREAM_PLAYBACK) ?
-		SOF_COMP_HOST : SOF_COMP_DAI;
+	fileread->rate = tp->fs_in;
+	fileread->channels = tp->channels_in;
+	fileread->frame_fmt = tp->frame_fmt;
+	fileread->direction = dir;
+
+	file_uuid = (struct sof_uuid *)((uint8_t *)fileread + sizeof(struct sof_ipc_comp_file));
+	file_uuid->a = 0xbfc7488c;
+	file_uuid->b = 0x75aa;
+	file_uuid->c = 0x4ce8;
+	file_uuid->d[0] = 0x9d;
+	file_uuid->d[1] = 0xbe;
+	file_uuid->d[2] = 0xd8;
+	file_uuid->d[3] = 0xda;
+	file_uuid->d[4] = 0x08;
+	file_uuid->d[5] = 0xa6;
+	file_uuid->d[6] = 0x98;
+	file_uuid->d[7] = 0xc2;
```

Comment:
how about a strcpy to replace above?

Reply:
Yes, could be better, or at least better readable.